compiler/rustc_codegen_ssa/src/back/linker.rs

```diff
@@ -835,6 +835,22 @@ impl<'a> Linker for GccLinker<'a> {
             if let Err(error) = res {
                 self.sess.dcx().emit_fatal(errors::LibDefWriteFailure { error });
             }
+        } else if crate_type == CrateType::Executable && !self.sess.target.is_like_solaris {
+            // Write an LD version script
+            let res: io::Result<()> = try {
+                let mut f = File::create_buffered(&path)?;
+                writeln!(f, "{{")?;
+                if !symbols.is_empty() {
+                    for (sym, _) in symbols {
+                        debug!("  {sym};");
```

Comment:
```suggestion
                        debug!(?sym);
```

don't need to format debug messages like the source